scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -118,6 +118,14 @@
     tags:
       - create-ardana-input-model
 
+  - name: Switch designate to use bind in Input Model
```

Comment:
please add a comment with a link to https://suse-jira.dyndns.org/browse/SCRD-2479